### ISSUE TITLE
Improve Micropython detection

### ISSUE THREAD

**Issue by ace-dent** [TinyCircuits/TinyCircuits-Thumby-Code-Editor#100]
Do not ask user to 'upgrade' Thumby Micropython, if the on device version exceeds that of the web IDE.

**Comment by jmtinycircuits** [comment]
Good point, this should be fixed in https://github.com/TinyCircuits/TinyCircuits-Thumby-Code-Editor/commit/016b8ff70778a0af5fcfe92ba33a3a2c3fc9e5f6

Feel free to close if you think my change makes sense (I did test it using MicroPython 1.18.0, 1.19.1, and 1.20.0 on a Thumby)
